prototype/README.md

```diff
@@ -0,0 +1,13 @@
+# Consultation Analyser Frontend Prototype
+
```

Comment:
Could stick this in the main README in case people don't think to look in the prototype folder - but just a suggestion.

Reply:
Good idea. I've done that now. Thanks Nina.